### ISSUE TITLE
Manage Access for Frameworks Bug

### ISSUE THREAD

**Issue by jfloyd23** [opensalt/opensalt#517]
Working with version OpenSALT 2.1.0

When we create a user and assign them to a framework through "Manage Access," they have no edit rights.

Use case: we want to allow users to edit existing frameworks without creating new frameworks. We created an account with no roles selected, and then we used "Manage Access" to grant access to a specific framework. However, the user was not able to edit the selected framework.

**Comment by brandonopened** [comment]
- Are the users editors?

**Comment by jfloyd23** [comment]
We were hoping to limit their editor abilities to a specific framework and saw no way to edit this access at the framework level. Currently the user has no role selected.


Our use case is that we would prefer users not create new frameworks or, more concerning, place frameworks incorrectly using the "Creator" field in the framework creation form.


Eric Floyd
Manager, Learning Technology Solutions
Center for Excellence in Teaching and Learning
Library South, Suite 100
103 Decatur St.
Atlanta, GA 30303
Georgia State University
jfloyd23@gsu.edu
404-413-4742
________________________________

**Comment by roverwolf** [comment]
Currently, adding a user to a framework through "Manage Access" does not change the user's role on that framework, it only gives them access to that framework with whatever role(s) they have.

Changing the roles per framework/organisation should be considered as we examine the overall "workflow" enhancements.

**Comment by jmarks** [comment]
Agreed, this part of workflows for 3.0. However, a work around is to create an org for each (set of) frameworks you want to give limited edit rights to. As long as a users does not need edit rights to 2 diferent sets, this should work. Super editors would still have edit rights to sll frameworks.

**Comment by roverwolf** [comment]
@jmarks That does not help with the main two issues in the use case:
- users not being able to create new frameworks
- users incorrectly using the "Creator" field
